lib/index.js

```diff
@@ -1 +1,121 @@
-// Your code will go here.
+"use strict";
+
+import md5 from "md5";
+import uuid from "uuid";
+import "isomorphic-fetch";
+
+var localStorage;
+if (typeof (localStorage) == "undefined" || localStorage === null) {
+  var LocalStorage = require('node-localstorage').LocalStorage;
+  localStorage = new LocalStorage('./lib/localStorage');
```

Comment:
So as discussed during this morning's standup, I think we should never try to conditionally require a fake localStorage implementation in browser land, as it might differs with the one of the browser, and would load a bunch of supplementary code never used as localStorage is available from quite all recent browser versions.

What we need to do here is to only load the fake localStorage implementation in node test environment, so the tests can use the API and perform assertions as if we were in a browser env.

This has been done in the past for the LocalStorage adapter in Kinto.js, where we loaded the localstorage node package only for the tests through a setup.js file which was required in the test setup procedure.

Usually that's matter of running the mocha test suite adding a `--require test/setup.js`, and in test/setup.js requiring the localstorage package and exposing its main localStorage object globally. From there you'll have a fake global localStorage implementation available from your test code.

Hope this helps.